ipatests/test_integration/test_uninstallation.py

```diff
@@ -55,6 +55,11 @@ def test_failed_uninstall(self):
         serverid = realm_to_serverid(self.master.domain.realm)
         instance_name = ''.join([dsinstance.DS_INSTANCE_PREFIX, serverid])
 
+        # Stop tracking the DS cert now to make it easier to clean up
+        # later.
+        ds = dsinstance.DsInstance()
+        ds.stop_tracking_certificates(None)
```

Comment:
This code will be executed with the local NSSDB of a local 389-DS instance, not the remote instance. That doesn't make much sense.

Reply:
Sigh, yes, this was me trying to be too clever by half. I removed a master invocation of ipa-getcert to replace this :-(  mostly because of hardcoding Server-Cert. Given it is a controlled environment I guess that's not a big deal.